code-challenges.test.js

```diff
@@ -15,46 +15,146 @@
 
+// var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
+// Expected output: [-823, 7, 23]
 
 // b) Create the function that makes the test pass.
-
-
+//Pseudo Code:
+// I need a function that takes in arrays  const sortThisOut  = [] ==>{}
+// I need that function to return only numbers using typeOf()
+// I need that function to return an array with only odds  using % 2 == 1
+// I need the function to sort the odd numbers in my array from least to greatest using Math.max()
+
+const sortThisOut = (array) => {
+    let newArr = []
+    for(let i=0; i<array.length; i++){
+      if(array[i] % 2 !== 0){
+        newArr.push(array[i])
+      } else if(array[i] !== typeof(String)){
+
+      }
+      
+    }
+    return newArr.sort((a,b) => a-b)
+  }
+  
+// const sortThisOut = (array) =>{
+// let newArray = []
+// for (let index = 0; index < array.length; index++) {
+//     if(array % 2 === 1){
+//         newArray.push(array)
+//     }
+// }
+// return newArray.typeOf(Number)
+// }
+// // console.log(sortThisOut(fullArr2));
+
+describe(sortThisOut, ()=> {
+    it('takes in an array and returns a new array of only odd numbers sorted from least to greatest', ()=> {
+    expect(sortThisOut(fullArr1)).toEqual([-9, 7, 9, 199])
+    })
+    })
 
 // --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-var numbersToAdd1 = [2, 4, 45, 9]
+// describe(iCannot, ()=> {
+//     it('is function that takes in an array and returns an array of the accumulating sum', ()=> {
+//     expect(iCannot(numbersToAdd1)).toEqual([2, 6, 51, 60])
+//     })
+//     })
+
+// var numbersToAdd1 = [2, 4, 45, 9]
 // Excpected output: [2, 6, 51, 60]
 
-var numbersToAdd2 = [0, 7, -8, 12]
-// Expected output: [0, 7, -1, 11]
+// var numbersToAdd2 = [0, 7, -8, 12]
+// // Expected output: [0, 7, -1, 11]
 
-var numbersToAdd3 = []
+// var numbersToAdd3 = []
 // Expected output: []
 
 
-
 // b) Create the function that makes the test pass.
+
+const iCannot = (num1) => {
+    let array =[];
+    for (let index = 2; index <= num1; index++) {
+        blip[i] = blip[i - 1] + blip[i - 2]
+        array.push(fib(i))
+        const element = array[index];  
+    }
+    return array.push(iCannot)
+}
+
+describe(iCannot, ()=> {
+    it('is function that takes in an array and returns an array of the accumulating sum', ()=> {
+    expect(iCannot(numbersToAdd1)).toEqual([2, 6, 51, 60])
+    })
+    })
+
+// //Psuedo Codding:
+// First I need to make a function that takes in an array
+// I need to much like the fibonacci sequence have the array go through a for loop
+// I need to iterate the numbers in the array through an equation that will add the current number to the number of the previous index. 
+// I need it to return in the form of an array.
```

Comment:
Okay I can see here you mostly copied the fib sequence math. That can sometimes throw us off really hard as developers. Pseudo coding this problem out in your own words and writing down all the logical steps will make this problem a lot easier on yourself.